lib/Messenger/ElementDependenciesMessage.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Messenger;
+
+use Pimcore\Model\Element\AbstractElement;
+
+/**
+ * @internal
+ */
+class ElementDependenciesMessage
+{
+    public function __construct(protected AbstractElement $element)
```

Comment:
I guess it does not work because of the lazy loading of relations. Therefore relation attributes are `null` directly after loading the object without calling the relation attribute getter.

Reply:
Fixed now.